target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -427,6 +427,10 @@ wavlink,wl-wn575a3)
 	ucidef_set_led_rssi "wifi-med" "wifi-med" "$boardname:green:wifi-med" "wlan1" "50" "84"
 	ucidef_set_led_rssi "wifi-high" "wifi-high" "$boardname:green:wifi-high" "wlan1" "85" "100"
 	;;
+wevo,11acnas|\
+wevo,w2914ns-v2)
+	ucidef_set_led_usbport "usb" "USB" "$boardname:green:usb" "usb1-port1" "usb1-port2" "usb2-port1"
```

Comment:
This seems redundant/overlapping with the triggers defined for the USB led. Do you need this to get back control after the diag.sh script was running?

Reply:
Yes, exactly.

Reply:
Okay, so this already shows why using the USB led for diag is not a good idea, as it requires "hacks" like this one. However, I'm not a core-developer, so I will leave this for other reviewers to decide.

Reply:
In my defense, there's already some devices with status led that goes back to the original role after boot. https://github.com/openwrt/openwrt/commit/0c5a53255307ab532b2f875db452803b7f297874 https://github.com/openwrt/openwrt/commit/a9360452f0130bb39a8e732334a174cc3f8991fd https://github.com/openwrt/openwrt/commit/9f0e2335764674d0324dd90adb3e185cc8befe5f

Reply:
Yes, and I would say it depends on the reviewer whether stuff like that comes in or not.

BTW, you provided an excellent choice of reference here, as each case is different:
0c5a532 uses LAN led without trigger in DTS
a936045 uses dts trigger for one led and 02_network for the other, but uses both for diag
9f0e233 uses signal strength indicator

Especially the last case has been accepted several times recently (particularly for the TP-Link CPExxx devices in ath79 target), as signal strenght indicators do not blink typically and thus have a reduced potential to confuse the user.
However, note that 9f0e233 uses the green indicator for diag AND signaling although the red one is completely unused. So this could also have slipped in ...

Nevertheless, I intentionally commented about this without a clear indication, as it depends on taste.

Reply:
BTW "usbport" trigger does not blink, the LED just turns on when USB devices inserted. That's why I thought USB LED could also be a nice status LED.

Reply:
No blinking on "activity"?

Reply:
Nope. https://www.kernel.org/doc/html/latest/leds/ledtrig-usbport.html
"disk-activity" might blink, but "usbport" does not have any blink option AFAIK.

Reply:
Okay, I'd say that this raises your chances :-)

If used as diag LED, one might consider removing the DTS trigger, so LED is only used for USB "after boot".

Reply:
Good point. Done.